docs/content/preview/releases/release-notes/v2.20.md

```diff
@@ -58,6 +58,165 @@ For more information, see [#20827](https://github.com/yugabyte/yugabyte-db/issue
+   </a>
+ </li>
+</ul>
+
+### Docker
+
+```sh
+docker pull yugabytedb/yugabyte:2.20.2.0-b145
+```
+
+### New features
+
+* Added support for read-only mode for DR replica, currently supported for YSQL tables.
+
+* Tablet splitting now can be enabled on tables with CDC configured. {{<issue 18479>}}
+
+* Transactional CDC now supports consistent snapshots, ensuring data integrity during replication to a sink. Snapshots, ordered by commit time across all tables and tablets, establish a reliable replication order. {{<issue 19682>}}
+
+
+<!-- #### DocDB
+
+* Enables a new API to filter Prometheus metric output, providing server-level aggregation for all tablet and table metrics. Users activate it with `version=v2` in the URL. The new setup introduces `server_allowlist`, `server_blocklist`, `table_allowlist`, `table_blocklist` regex filters for greater control. {{<issue 19943>}}
+* Temporarily adjusts the GFlag to not set a limit on the per-tablet overhead memory tracker, enabling the use of the tablet guardrail mechanism without previous issues and allowing for a safer control over the number of tablets created. {{<issue 20667>}}
+
+#### CDC
+
+* Enhances CDCSDK to communicate tablet splits to clients as soon as they are detected during `GetChanges` calls, offering improved data handling via hash_key checks and facilitating modifications to tests. {{<issue 18479>}}
+* Adds yb-admin CLI support for creating consistent snapshot streams with options NOEXPORT_SNAPSHOT and USE_SNAPSHOT, guarded by the TEST flag yb_enable_cdc_consistent_snapshot_streams. {{<issue 19682>}}
+* Sets the CDC Consistent Snapshot feature as a preview with `USE_SNAPSHOT` as the default consistent snapshot option while creating a stream, and guards changes by the PREVIEW flag `yb_enable_cdc_consistent_snapshot_streams`. {{<issue 20367>}} -->
+
+### Improvements
+
+#### YSQL
+
+* Introduces yb_silence_advisory_locks_not_supported_error as a temporary solution, avoiding disruption while users transition from the use of advisory locks. {{<issue 19974>}}
+* Shifts from the test gflag "FLAGS_TEST_enable_db_catalog_version_mode" to the tech preview gflag "FLAGS_ysql_enable_db_catalog_version_mode", enhancing user control over concurrent DDL execution across different databases. {{<issue 12417>}}
+* Issues a notice for unsafe ALTER TABLE operations, including for ADD COLUMN...DEFAULT, to indicate existing rows won't be backfilled with the default value, enhancing user awareness. Suppression possible by setting ysql_suppress_unsafe_alter_notice gflag to true. {{<issue 19360>}}
+* Added sorting capabilities to BatchNestedLoopJoin to return the rows in the same order as NestedLoopJoin {{<issue 19589>}}
+* Replaces the `ysql_max_read_restart_attempts` and `ysql_max_write_restart_attempts` gflags with `yb_max_query_layer_retries`, applies limit to Read Committed isolation statement retries, and adjusts `retry_backoff_multiplier` and `retry_min_backoff` defaults. {{<issue 20359>}}
```

Comment:
here the flags are in code - why not everywhere?

Reply:
Because this gflag is not documented at all.